nss-day-cohort-60/rare-client-tenacious-tators
# Jes styling and tags

# Description of Proposed Changes

Styling added, and multiple tags displayed

Ive tested all functionality on this branch and it all works on my end. 



---


# Steps to Test

1. Fetch the jes-styling-and-tags branch and switch to it
2. navigate through the site and make sure nothing is broken
3.
